RJDBC/pom.xml

```diff
@@ -19,12 +19,24 @@
 			<groupId>org.renjin.cran</groupId>
 			<artifactId>DBI</artifactId>
 			<version>0.7-b4</version>
+			<!-- TODO: maybe upgrade to a later version e.g. 1.0.0-b9 -->
 		</dependency>
 		<dependency>
 			<groupId>org.renjin</groupId>
 			<artifactId>gcc-runtime</artifactId>
 			<version>${renjin.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>joda-time</groupId>
+			<artifactId>joda-time</artifactId>
+			<version>2.10.1</version>
+		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
```

Comment:
Why add a dependency on h2database to RJDBC?
We should have a separate module for h2.

Reply:
This was to verify that the addition of TIMESTAMP worked. It was just a test dependency. To avoid confusion on "why is this not in h2 module" I changed the test to use Apache Derby instead. Hope that makes it clearer.